ksp_plugin.py

```diff
@@ -4,13 +4,16 @@
 import codecs
 import traceback
 import os.path
-from   os import listdir
+import os
+from pathlib import Path
```

Comment:
Will need branching by ST version because this `pathlib` is not available in ST3.